lib/usd/ui/layerEditor/saveLayersDialog.cpp

```diff
@@ -513,19 +513,26 @@ void SaveLayersDialog::onCancel() { reject(); }
 
 bool SaveLayersDialog::okToSave()
 {
-    int         i, count;
-    QStringList existingFiles;
+    // Files can have the same file names in complicated ways, with one files having two copies,
+    // another three, so we keep the exact number of copies per file path.
+    std::map<QString, int> alreadySeenPaths;
```

Comment:
I probably would have used a QMap here just to stay in Qt (vs std). The Qt containers are better optimized for handling Qt classes.